src/core/types/coreTypes.js

```diff
@@ -0,0 +1,6 @@
+/* @flow */
```

Comment:
Do these filenames all need `Types` as suffixes? They're already in a folder named `types`–can we nix the `Types` suffix?